sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -421,10 +502,6 @@ class HealthcareEntity(DictMixin):
     :ivar int offset: The entity text offset from the start of the document.
```

Comment:
Also, almost forgot -- I think we also need to add `normalized_text` as a property (renamed from `name` in the original result).

Reply:
yeah, doing that in a separate pr, wanted to keep this as much just "relations" as possible